pkg/sql/alter_role.go

```diff
@@ -79,8 +79,9 @@ func (p *planner) AlterRoleNode(
 	kvOptions tree.KVOptions,
 ) (*alterRoleNode, error) {
 	// Note that for Postgres, only superuser can ALTER another superuser.
-	// CockroachDB does not support superuser privilege right now.
-	// However we make it so the admin role cannot be edited (done in startExec).
```

Comment:
Wait I'm curious, so this comment wasn't true before since admins could be altered?

Reply:
the comment was true, but it was a kind of silly limitation. we only prevented editing ADMIN role directly (see line 166)

Reply:
Ah I see okay so the literal "admin" role cannot and still cannot be edited.